prime-router/src/main/kotlin/transport/SftpTransport.kt

```diff
@@ -43,7 +43,16 @@ class SftpTransport : ITransport, Logging {
             if (header.content == null)
                 error("No content to sftp for report ${header.reportFile.reportId}")
             val receiver = header.receiver ?: error("No receiver defined for report ${header.reportFile.reportId}")
-            val credential = lookupCredentials(receiver.fullName)
+
+            // if the transport definition has defined default
+            // credentials use them, otherwise go with the
+            // standard way by using the receiver full name
+            val credential = if (sftpTransportType.credentials != null) {
+                getCredentials(sftpTransportType.credentials)
+            } else {
+                lookupCredentials(receiver.fullName)
+            }
+
```

Comment:
I am concerned about the order of operations here because it's possible that someone could accidentally assign default credentials to a receiver and that would take precedent over the already defined credentials.  I think it would be better for it to fallback to the default if the credentials for the receiver name don't exist.

Reply:
I disagree. The downside would be a failed login to an SFTP site. I think if we're explicitly saying _use these credentials_ we should follow what we define, as if it's an override.

Reply:
I'll be the tiebreaker vote, in this case to adopt Ron's lookupCredentials suggestion, above.  

Note: The default credentials feature should only be a dev/test feature.

Reply:
fixed